modules/mono/glue/GodotSharp/GodotPlugins/Main.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
```

Comment:
Style issue noted on this line, probably means that there is a BOM? https://github.com/godotengine/godot/pull/52610/checks?check_run_id=3580693511